index.html

```diff
@@ -554,6 +554,775 @@ <h2>OpenID Connect</h2>
+key agreement protocols with the <a>controller</a>&mdash;the value of the
+`<a>keyAgreement</a>` property needs to be used for that.
+          </p>
+          <p>
+The <a>controller document</a> does not express revoked keys using a <a>verification
+relationship</a>. If a referenced verification method is not in the latest
+<a>controller document</a> used to dereference it, then that verification method is
+considered invalid or revoked.
+          </p>
+          <p>
+The following sections define several useful <a>verification relationships</a>.
+A <a>controller document</a> MAY include any of these, or other properties, to
+express a specific <a>verification relationship</a>. In order to maximize global
+interoperability, any such properties used SHOULD be registered in the
+Data Integrity Specification Registries [TBD: DIS-REGISTRIES].
+          </p>
+
+          <section>
+            <h2>Authentication</h2>
+
+            <p>
+The `authentication` <a>verification relationship</a> is used to
+specify how the <a>controller</a> is expected to be <a>authenticated</a>, for
+purposes such as logging into a website or engaging in any sort of
+challenge-response protocol.
+            </p>
+
+            <dl>
+              <dt id="defn-authentication">authentication</dt>
+              <dd>
+The `authentication` property is OPTIONAL. If present, the associated
+value MUST be a <a data-cite="INFRA#ordered-set">set</a> of one or more
+<a>verification methods</a>. Each <a>verification method</a> MAY be embedded or
+referenced.
+              </dd>
+            </dl>
+
+            <pre class="example nohighlight" title="Authentication property
+                          containing three verification methods">
+    {
+      "@context": [
+        "https://www.w3.org/ns/did/v1",
+        "https://w3id.org/security/multikey/v1"
+      ],
+      "id": "did:example:123456789abcdefghi",
+      <span class="comment">...</span>
+      "authentication": [
+        <span class="comment">// this method can be used to authenticate as did:...fghi</span>
+        "did:example:123456789abcdefghi#keys-1",
+        <span class="comment">// this method is *only* approved for authentication, it may not</span>
+        <span class="comment">// be used for any other proof purpose, so its full description is</span>
+        <span class="comment">// embedded here rather than using only a reference</span>
+        {
+          "id": "did:example:123456789abcdefghi#keys-2",
+          "type": "Multikey",
+          "controller": "did:example:123456789abcdefghi",
+          "publicKeyMultibase": "z6MkmM42vxfqZQsv4ehtTjFFxQ4sQKS2w6WR7emozFAn5cxu"
+        }
+      ],
+      <span class="comment">...</span>
+    }
+            </pre>
+
+            <p>
+If authentication is established, it is up to the application to decide what to
+do with that information.
+            </p>
+            <p>
+This is useful to any <em>authentication verifier</em> that needs to check to
```

Comment:
This section seems to need integration with the latest upstream data model text regarding holder authentication